sabnzbd/filesystem.py

```diff
@@ -58,6 +58,23 @@ def get_ext(filename: str) -> str:
         return ""
 
 
+def has_unwanted_extension(filename: str) -> bool:
+    """ Determine if a filename has an unwanted extension, given the configured mode """
+    extension = get_ext(filename).replace(".", "").lower()
```

Comment:
`get_ext` is always lower-cased, I did think of that 😉

Reply:
I should have notice, I put the new function right nextdoor...